### PR TITLE
Fix `titleLabel` font upgrade for `SSDynamicButton` after font change…

### PR DESCRIPTION
**Pull request by Grubas7**
Fix for issue #27.

### CODE REVIEW

**Comment by Grubas7** [comment]
@jhersh Ready to review if you have time ;)

File: SSDynamicText/UIView+SSTextSize.h [hunk 1]
```diff
@@ -27,28 +27,35 @@ typedef void (^SSTextSizeChangedBlock) (NSInteger);
...
  */
-- (void) preferredContentSizeDidChange;
+- (void)preferredContentSizeDidChange;
 
 /**
  * Default FontName if set in Info.plist or systemFontName if not set
  * Key: kSSDynamicDefaultFontName
  */
-- (NSString *) ss_defaultFontName;
+- (NSString *)ss_defaultFontName;
```
Comment (jhersh) [inline SSDynamicText/UIView+SSTextSize.h]: This and `ss_defaultBaseSize` could be `readonly` `@property`s.

**Comment by jhersh** [comment]
:+1:

File: SSDynamicText/UIView+SSTextSize.h (hunk 1 above)
Comment (Grubas7) [inline SSDynamicText/UIView+SSTextSize.h]: I would like to add nullability and generics, andI  would change this to property in same PR.
Is that OK for you?

Comment (jhersh) [inline SSDynamicText/UIView+SSTextSize.h]: Nice!

Comment (Grubas7) [inline SSDynamicText/UIView+SSTextSize.h]: I merge this PR, and I'll create new PR over a days with generics, nullability, this property and some minors improvements ;)

**Pull request merged**